djangoapp/app/views.py

```diff
@@ -0,0 +1,52 @@
+import os, platform, configparser
+from django.shortcuts import render
+from .forms import ProfileForm
+from .models import ProfilesTable
+from django_tables2 import RequestConfig
+import boto3
+import botocore
+
+def index(request):
+  BUCKET_NAME = 'cloud-platform-reference-bucket'
+  FILE_NAME = 'hello.txt'
+  with open(FILE_NAME, 'rb') as f:
+    fstr = f.read()
+  if request.method == 'POST':
+    pass
+  else:
+    if (os.getppid() == 1):
+      osname = 'Docker'
+      table = ProfilesTable()
+    else:
+      osname = platform.system()
+      awsconfig = configparser.ConfigParser()
+      awsconfig.read(os.path.expanduser('~/.aws/credentials'))
```

Comment:
We shouldn't be using the AWS creds file here - creds should be defined as environment variables, or in the case of AWS creds possibly externally as IAM Roles.

Reply:
this was on purpose, will detail